src/internal/components/dropdown/index.tsx

```diff
@@ -217,6 +221,26 @@ const Dropdown = ({
     }
   };
 
+  const focusHandler = (event: React.FocusEvent) => {
+    const previouslyFocusedElement = event.relatedTarget;
+    if (
+      (!wrapperRef.current || !wrapperRef.current.contains(previouslyFocusedElement)) &&
+      (!dropdownContainerRef.current || !dropdownContainerRef.current.contains(previouslyFocusedElement))
```

Comment:
could this condition be extracted to a function with a name that describes what it checks for?

```suggestion
      const isFocusTrapped = (wrapper, dropdown, focusedElement) => 
       (!wrapper.current || !wrapper.current.contains(focusedElement)) &&
      (!dropdown.current || !dropdown.current.contains(focusedElement))
```